tests/unit/test_aiopg_connector.py

```diff
@@ -68,18 +68,21 @@ async def corofunc(connector):
 
 
 @pytest.mark.asyncio
+@pytest.mark.parametrize("pool_max_size", [5, "5"])
 @pytest.mark.parametrize(
     "exception_class", [Exception, psycopg2.errors.OperationalError]
 )
```

Comment:
I'm not sure I understand why you're doing that.

Do we really need to test the cartesian product of the pool sizes and the exception classes here ?

Reply:
Without this line, the test never pass in the following exception https://github.com/peopledoc/procrastinate/blob/master/procrastinate/aiopg_connector.py#L64.

You are right, we do not need to test the full cartesian product. Maybe we can reach that exception in an other test.

Reply:
I put it in a more relevant test, see upgrade. I let you close this discussion if you are ok with the upgrade :)

Reply:
Yep, when possible, I prefer to have:
- the shortest tests
- tests that test the least amount of code
- the least amount of tests

Of course, there's a balance that needs to be struck between those 3, but adding 2 complex tests to cover a single missing line seems improvable :)

Reply:
Improved in an other test using parametrize. It is much cleaner.